packages/xchain-ethereum/src/client.ts

```diff
@@ -717,21 +660,22 @@ export default class Client implements XChainClient, EthereumClient {
    * @throws {"Invalid Asset Address"}
    * Thrown if the given asset address is invalid.
    */
-  erc20Tx = async ({ assetAddress, recipient, amount, overrides }: Erc20TxOpts): Promise<TransactionResponse> => {
+  erc20Transfer = async ({ assetAddress, recipient, amount, overrides }: Erc20TxOpts): Promise<TxHash> => {
```

Comment:
Do not have a separate public function for a ERC20-tx. The upstream wallet should not care if transferring ETH or ERC20, it is this client's job to detect which asset and proceed accordingly. 

Simply have one master `transfer()` function.